rune/libenclave/internal/runtime/pal/skeleton/Makefile

```diff
@@ -67,8 +67,14 @@ else
         PRODUCT_OPT :=
 endif
 
+ifdef MEMORY_SIZE
+MEMORY_OPT := -m $(MEMORY_SIZE)
+else
+MEMORY_OPT :=
+endif
+
```

Comment:
You don't need to ask for inputing a static memory size during build. Instead, allowing sgx_sign tool to specify -m is enough.